### ISSUE TITLE
Gradle cannot execute buildHelpFiles on Windows

### ISSUE THREAD

**Issue by esaulenka** [NationalSecurityAgency/ghidra#7034]
**Describe the bug**

Cannot build help files from the actual master branch.
Environment - Win10 + Msys2

**To Reproduce**
1. do `git pull`
2. run `gradle buildGhidra`

```
$ gradle buildGhidra

> Configure project :
Java: 21
Gradle: 8.10.1
Python3 command: [py] (version 3.12)
Python3 pip version: 24.0
OS: Windows 10
Arch: amd64 (gradle: x86-64)
Using platform: win_x86_64
Searching for latest Visual Studio and required components...
  -> Installation Directory: C:\Program Files (x86)\Microsoft Visual Studio\2017\Community
  -> SDK Directory (default): C:\Program Files (x86)\Windows Kits\10\
  -> SDK Version (default): 10.0.17763.0
  -> SDK Version (override): N/A

> Task :Base:buildHelpFiles
Exception in thread "main" java.nio.file.InvalidPathException: Illegal char <:> at index 8: external:docs/WhatsNew.html
        at java.base/sun.nio.fs.WindowsPathParser.normalize(WindowsPathParser.java:204)
        at java.base/sun.nio.fs.WindowsPathParser.parse(WindowsPathParser.java:175)
        at java.base/sun.nio.fs.WindowsPathParser.parse(WindowsPathParser.java:77)
        at java.base/sun.nio.fs.WindowsPath.parse(WindowsPath.java:92)
        at java.base/sun.nio.fs.WindowsFileSystem.getPath(WindowsFileSystem.java:231)
        at java.base/java.nio.file.Path.of(Path.java:148)
        at java.base/java.nio.file.Paths.get(Paths.java:69)
        at help.validator.LinkDatabase.getIDForLink(LinkDatabase.java:181)
        at help.validator.model.TOCItemDefinition.validate(TOCItemDefinition.java:39)
        at help.validator.JavaHelpValidator.validateTOCItemIDs(JavaHelpValidator.java:358)
        at help.validator.JavaHelpValidator.validate(JavaHelpValidator.java:397)
        at help.GHelpBuilder.validateHelpDirectories(GHelpBuilder.java:140)
        at help.GHelpBuilder.build(GHelpBuilder.java:95)
        at help.GHelpBuilder.main(GHelpBuilder.java:85)

> Task :Base:buildHelpFiles FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':Base:buildHelpFiles'.
> Process 'command 'C:\Program Files\Eclipse Adoptium\jdk-21.0.3.9-hotspot\bin\java.exe'' finished with non-zero exit value 1
```

Looks like this `external:` prefix in the filenam breaks compilation:
https://github.com/NationalSecurityAgency/ghidra/commit/81212432a327560317f0fe3522221499b8bbf2bb#diff-377819e5026067fc8befaad4569d23152ed7f58f986d07928a841bd4f0ec6c20R382

Switching to the commit before this one fixes the issue.

`gradle buildGhidra -x buildHelpFiles` works fine as well.

**Environment (please complete the following information):**
 - OS: win10
 - Java Version: 21
 - Ghidra Version: ce6bbe6236a191830dfb50d4a549c5fd23de06ae
 - Ghidra Origin: github latest master

**Comment by ryanmkurtz** [comment]
Can you pull `master` and try again?

**Comment by esaulenka** [comment]
**BUILD SUCCESSFUL**

thanks @ryanmkurtz, thanks @dragonmacher !

**Comment by ryanmkurtz** [comment]
Fixed by 29fe2d8b17a6d0f5cf2e1a95a51510928820de58
